attachment_queue/models/attachment_queue.py

```diff
@@ -73,6 +73,9 @@ def _compute_failure_emails(self):
 
     def _get_failure_emails(self):
         # to be overriden in submodules implementing the file_type
+        _logger.info(
```

Comment:
suggestion for deprecation

https://github.com/odoo/odoo/blob/83a7daa07fedcaeddb983a94a86e4c7a187915d9/odoo/tools/misc.py#L86

LGTM